### PR TITLE
UI is not updating when user email is verified in EmailPasswordActivity

### PR DESCRIPTION
**Pull request by VikasKumarPatel**
each time when EmailPasswordActivity Start verify button is showing even wehn email is verified
IMG_20210120_124957

### CODE REVIEW

File: auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt [hunk 1]
```diff
@@ -344,7 +344,12 @@ class PhoneAuthActivity : AppCompatActivity(), View.OnClickListener {
 
                 verifyPhoneNumberWithCode(storedVerificationId, code)
             }
-            R.id.buttonResend -> resendVerificationCode(binding.fieldPhoneNumber.text.toString(), resendToken)
+            R.id.buttonResend ->  {
```
Comment (samtstern) [inline auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt:347]: Are these changes related to this fix?

**Review by samtstern — CHANGES_REQUESTED** [review]

**Comment by samtstern** [comment]
@VikasKumarPatel thanks for the PR. This does seem like a bug but can you explain a little more why your fix is the right solution?

**Review by VikasKumarPatel — COMMENTED** [review]
Changes in PhoneAuthActivity is already done
see ISSUE: 73b077d42415ffb3a0040a23e8ff062d0d4080e1 and ff907a0e88de66ccbcb453e2c1dfb2b89914750f

File: auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt (hunk 1 above)
Comment (pathflores) [reply auth/app/src/main/java/com/google/firebase/quickstart/auth/kotlin/PhoneAuthActivity.kt:347]: #👋-General#😁-General

**Review by pathflores — COMMENTED** [review]
https://support.google.com/websearch/thread/88117769?hl=en&msgid=88117769

**Review by VikasKumarPatel — COMMENTED** [review]
changes to PhoneAuthActivity is removed, kindly review changes

**Review by samtstern — APPROVED** [review]

**Comment by samtstern** [comment]
@VikasKumarPatel LGTM thank you!

**Pull request merged**